src/app/story/storyrow/storyrow.component.html

```diff
@@ -1,11 +1,16 @@
 <div class="node-uid-column story-row-column">
-  <div class="node-display-name">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
+  <div class="node-display-name lengthened">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
+  <div class="node-display-name shortened">{{ node.display_name | nodeDisplayName | truncate: 15 }}</div>
   <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">
     {{ node.timeline_timestamp | date: 'MMM d, y, HH:mm:ss':'+0000' }} UTC
   </div>
   <ng-template class="node-timestamp" #noTimestamp>
     <div class="node-timestamp">No timestamp available</div>
   </ng-template>
+  <div class="wait-data" *ngIf="!last">
+    <div class="wait-time">{{ waitTime | timeDisplay:true }}</div>
```

Comment:
This doesn't seem to always lineup correctly. Look at the second one.
![image](https://user-images.githubusercontent.com/11711106/62727415-6daaba00-b9e7-11e9-86d9-2bca9503ba44.png)